kvbc/src/categorization/block_merkle_category.cpp

```diff
@@ -18,38 +18,38 @@
 #include "assertUtils.hpp"
 #include "kv_types.hpp"
 #include "sha_hash.hpp"
+#include <iostream>
```

Comment:
Nit: Do we need it?

Reply:
nope. We don't. I was printf debugging. I need to add a proper logger.